src/components/mainNavbar/mainNavbar.module.scss

```diff
@@ -1,5 +1,7 @@
 @import url("https://fonts.googleapis.com/css?familly=Poppins:200,300,400,500,600,700,800,900&display=swap");
```

Comment:
Refused to apply style from 'https://fonts.googleapis.com/css?familly=Poppins:200,300,400,500,600,700,800,900&display=swap' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.

What is that?